### PR TITLE
HERITAGE-326 Add styles for no results messaging

### PR DESCRIPTION
**Pull request by garethmead**
Ticket URL: [HERITAGE-326](https://national-archives.atlassian.net/browse/HERITAGE-326)

## About these changes

Add styling for new "No results" messaging

## How to check these changes

Ticket linked provides 3 different scenarios for showing no results

## Before assigning to reviewer, please make sure you have

- [x] Checked things thoroughly before handing over to reviewer
- [x] Checked PR title starts with ticket number as per project conventions to help us keep track of changes
- [x] Ensured that PR includes only commits relevant to the ticket
- [x] Waited for all CI jobs to pass before requesting a review
- [ ] Added/updated tests and documentation where relevant

## Merging PR guidance

Follow [docs\developer-guide\contributing.md](https://nationalarchives.github.io/ds-wagtail/developer-guide/contributing/)

## Deployment guidance

Follow [docs\infra\environments.md](https://nationalarchives.github.io/ds-wagtail/infra/environments/)

[HERITAGE-326]: https://national-archives.atlassian.net/browse/HERITAGE-326?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: templates/search/blocks/no_results.html
Comment (TNA-Allan) [inline templates/search/blocks/no_results.html:13]:
```suggestion
            <li>Use fewer keywords</li>
```

**Review by TNA-Allan — CHANGES_REQUESTED** [review]

**Review by TNA-Allan — APPROVED** [review]
Thanks @garethmead Looks good from BE perspective. Will wait for a FE approval.

**Review by ahosgood — APPROVED** [review]

**Pull request merged**
